docker/pom.xml

```diff
@@ -38,55 +38,65 @@
         <jakarta.authentication-api.version>2.0.0</jakarta.authentication-api.version>
         <!-- Jakarta Messaging -->
         <jms-api.version>3.0.0</jms-api.version>
-        <jakarta.activation-api.version>2.1.0-RC2</jakarta.activation-api.version>
+        <jakarta.activation-api.version>2.1.0</jakarta.activation-api.version>
         <!-- Jakarta Persistence -->
         <jakarta-persistence-api.version>3.1.0-RC1</jakarta-persistence-api.version>
         <!-- Jakarta Transactions -->
         <jakarta.transaction-api.version>2.0.0</jakarta.transaction-api.version>
         <!-- Jakarta Connectors -->
         <jakarta.resource-api.version>2.0.0</jakarta.resource-api.version>
         <!-- Jakarta Batch -->
-        <jakarta.batch-api.version>2.0.0</jakarta.batch-api.version>
+        <jakarta.batch-api.version>2.1.0-M1</jakarta.batch-api.version>
         <!-- Jakarta Enterprise beans -->
         <jakarta.ejb-api.version>4.0.0</jakarta.ejb-api.version>
         <!-- Jakarta JSON -->
         <jsonp-api.version>2.1.0</jsonp-api.version>
         <json.bind-api.version>3.0.0-SNAPSHOT</json.bind-api.version>
         <!-- Jakarta Server Pages -->
-        <jsp-api.version>3.0.0</jsp-api.version>
+        <jsp-api.version>3.1.0</jsp-api.version>
         <!-- Jakarta Standard Tag Library -->
         <jstl-api.version>2.0.0</jstl-api.version>        
         <!-- CDI -->
         <cdi-api.version>3.0.0</cdi-api.version>
         
         <jakarta.annotation-api.version>2.1.0</jakarta.annotation-api.version>
         <jakarta.el-api.version>5.0.0</jakarta.el-api.version>
-        <jakarta.inject-api.version>2.0.0</jakarta.inject-api.version>
+        <jakarta.inject-api.version>2.0.1</jakarta.inject-api.version>
         <jakarta.mail-api.version>2.1.0</jakarta.mail-api.version>
         <jakarta.servlet-api.version>6.0.0</jakarta.servlet-api.version>
-        <jakarta.validation-api.version>3.0.0</jakarta.validation-api.version>
+        <jakarta.validation-api.version>3.0.2-SNAPSHOT</jakarta.validation-api.version>
 
-        <jakarta.ws.rs-api.version>3.0.0</jakarta.ws.rs-api.version>
-        <jakarta.xml.bind-api.version>4.0.0-RC2</jakarta.xml.bind-api.version>
+        <jakarta.ws.rs-api.version>3.1.0</jakarta.ws.rs-api.version>
+        <jakarta.xml.bind-api.version>4.0.1-SNAPSHOT</jakarta.xml.bind-api.version>
         <jaxb-osgi.version>3.0.0</jaxb-osgi.version>
-        <!-- Jakarta XML Web Services 4.0 via jakarta.xml.ws:jakarta.xml.ws-api:jar:4.0.0* -->
-        <webservices-api.version>4.0.0-RC1</webservices-api.version>
-        <soapwithattachments-api.version>3.0.0-RC1</soapwithattachments-api.version>
-        <webservices-api-osgi.version>3.0.0</webservices-api-osgi.version>
+        <!-- Jakarta XML Web Services 4.0 via jakarta.xml.ws:jakarta.xml.ws-api:jar:4.0.1-SNAPSHOT* -->
+        <webservices-api.version>4.0.1-SNAPSHOT</webservices-api.version>
```

Comment:
@lukasj - shall we use 4.0.0-RC2 from https://jakarta.oss.sonatype.org/content/repositories/staging/jakarta/xml/ws/jakarta.xml.ws-api/ (4.0.1-SNAPSHOT has latest timestamp)

Reply:
yes, use the RC2 build. SNAPSHOT will almost always have the latest timestamp since it is being built after every commit